### ISSUE TITLE
Daily challenge chat can be opened in normal chat window

### ISSUE THREAD

**Issue by Oreeeee** [ppy/osu#29192]
### Type

Game behaviour

### Bug description

You can open the daily challenge chat in a normal chat window. I don't really think it was intentional, because the channel name shows as #lazermp-XXXXXX

Steps to reproduce:
1. Make someone mention you in daily challenge chat.
2. If you are in the daily challenge menu, a notification will appear about being mentioned.
3. (Optional) You can quit the daily challenge menu.
4. Click the notification.
5. Daily challenge chat will open in a normal chat window.

Additionally, the mention notification will appear every time you open the daily challenge menu, until the mention gets pushed off by other messages.

### Screenshots or videos

![osu_2024-07-30_00-30-39](https://github.com/user-attachments/assets/c0526c26-5b35-4279-9d27-990f4362198a)

### Version

2024.727.0-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16419735/compressed-logs.zip)

**Comment by Joehuu** [comment]
> Additionally, the mention notification will appear every time you open the daily challenge menu, until the mention gets pushed off by other messages.

That is already tracked in https://github.com/ppy/osu/issues/24715. Please only report one issue per thread.

**Comment by bdach** [comment]
This is not new or exclusive to daily challenge. Probably relevant: https://github.com/ppy/osu/issues/22279

**Comment by Oreeeee** [comment]
Yeah, this seems to be an issue with the UI element from which multiplayer, playlists and daily challenge menus inherit from.

**Comment by cloudrac3r** [comment]
I think it would be cool if I could keep up with the daily challenge chat from the regular chat screen.

**Comment by peppy** [comment]
I don't really see this as an issue.
